libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -108,8 +111,8 @@ class AppPrompt {
 
   // TODO: should we allow for app icon to be set as SVG?
   decorate = () => {
-    this.elements.closeIcon = toFragment`<button class="appPrompt-close"></button>`;
-    this.elements.cta = toFragment`<button class="appPrompt-cta appPrompt-cta--close">${this.cancelText}</button>`;
+    this.elements.closeIcon = toFragment`<button daa-ll="Close Modal" class="appPrompt-close"></button>`;
+    this.elements.cta = toFragment`<button daa-ll="${this.cancelText}" class="appPrompt-cta appPrompt-cta--close">${this.cancelText}</button>`;
```

Comment:
Is analytics looking for the translated text or the English one? Usually it's the latter they want, so placeholders might not be the right approach here.

Reply:
There was no specific requirement for translated text, so I changed it to the English version, which makes more sense 👍. I initially opted for the translated version because I observed this approach being used in some places in the global navigation.